src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/supported-languages.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum SupportedLanguages {
+  'en-US' = 'English',
+  'de' = 'Deutsch',
```

Comment:
Do you still plan to switch to the primary/extended subtag schema, e.g. using `de-DE` instead of `de`?

Reply:
Done